partner_email_check/models/res_config_settings.py

```diff
@@ -0,0 +1,43 @@
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    partner_email_check_filter_duplicates = fields.Boolean(
+        string="Filter duplicate partner email addresses",
+        help="Don't allow multiple partners to have the same email address.",
+    )
+
+    partner_email_check_check_deliverability = fields.Boolean(
+        string="Check deliverability of email addresses",
+        help="Don't allow email addresses with providers that don't exist",
```

Comment:
```suggestion
        help="Don't allow email addresses with providers that don't exist",
        config_parameter="partner_email_check_check_deliverability"
```
and trash get/set_values overrides ;)

BTW IMO this should be configured on the company, then you can get rid of the params.